README.md

```diff
@@ -5,6 +5,8 @@
 
 This machine charm creates self-hosted runners for running GitHub Actions. Each unit of this charm will start a configurable number of OpenStack or LXD based virtual machines to host them. Every runner performs only one job, after which it unregisters from GitHub to ensure that each job runs in a clean environment.
 
+To host local LXD based virtual machines, you have to use the track `local-lxd`. See [juju channels](https://discourse.charmhub.io/t/channel/6562) for more information on channels.
```

Comment:
What may be confusing is that the tutorial linked in the readme assumes an lxd machine. This means that the tutorial won't work if the lxd track is not used. So I think we need to either

- update the tutorial to use the lxd track channel 
- don't mention the tutorial here
- update the tutorial to use the microstack

Reply:
yeah, I will update all of them in the main branch when lxd is removed there

Reply:
yeah, good point. As this PR is to be merged in local-lxd, I have put the channel in the tutorial (and in the readme).

In the main branch, once lxd code is removed, everything will have to be adjusted

Reply:
besides, I will have to create another PR after this one (once there is a charm in the local-lxd track) to update the integration test to get the charm for the track (the integration test in test_charm_upgrade.py)